_includes/pbc/all/install-features/202304.0/install-the-packaging-units-feature.md

```diff
@@ -0,0 +1,1098 @@
+| ProductPackagingUnitDataImportPlugin | Imports packaging unit type data into the database. | <ul><li>Requires related product concretes and product abstract to be present in the database already.</li><li>Requires related packaging unit types to be present in the database already.</li></ul> | Spryker\Zed\ProductPackagingUnitDataImport\Communication\Plugin\DataImport |
+
+**src/Pyz/Zed/DataImport/DataImportDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\DataImport;
+
+use Spryker\Zed\DataImport\DataImportDependencyProvider as SprykerDataImportDependencyProvider;
+use Spryker\Zed\ProductPackagingUnitDataImport\Communication\Plugin\DataImport\ProductPackagingUnitDataImportPlugin;
+
+class DataImportDependencyProvider extends SprykerDataImportDependencyProvider
+{
+    /**
+     * @return list<\Spryker\Zed\DataImport\Dependency\Plugin\DataImportPluginInterface>
+     */
+    protected function getDataImporterPlugins(): array
+    {
+        return [
+            new ProductPackagingUnitDataImportPlugin(),
+        ];
+    }
+}
+```
+
+Run the following console command to import data:
+
+```bash
+console data:import product-packaging-unit
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the configured data in the database has been added to the `spy_product_packaging_unit`  and `spy_product_packaging_unit_type` tables.
+
+{% endinfo_block %}
+
+### 7) Set up behavior
+
+Set up the following behaviors.
+
+#### Set up checkout workflow
+
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN                                                       | SPECIFICATION                                                                                                                                    | PREREQUISITES                                                                                                               | NAMESPACE                                                               |
+|--------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------|
+| AmountAvailabilityCartPreCheckPlugin                         | Validates if the given amount is available according to stock configuration during the cart change.                                              | Expects the `amount` field to be set in `ItemTransfers` with packaging units.                                               | Spryker\Zed\ProductPackagingUnit\Communication\Plugin\Cart              |
```

Comment:
We have to add  the ProductPackagingUnitApiPickingListItemsAttributesMapperPlugin.